### PR TITLE
Add rules for ANSSI R12 - partitioning type

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:
- Select rules from cac master
  - Some rules are created in this PR: ComplianceAsCode#6340

### CODE REVIEW

**Comment by yuumasato** [comment]
This can be reviewed and merged.

File: controls/anssi.yml [hunk 1]
```diff
@@ -162,34 +162,52 @@ controls:
     level: intermediary
     title: Partitioning type
     rules:
+    # this covers nodev options
+    - mount_option_nodev_nonroot_local_partitions
```
Comment (jan-cerny) [inline controls/anssi.yml:166]: This is clever!

File: controls/anssi.yml [hunk 2]
```diff
@@ -162,34 +162,52 @@ controls:
...
+    - mount_option_srv_nosuid
+
     # /home nosuid, nodev, noexec Contains the HOME users.  Read-only editing if not in use
     - partition_for_home
     - mount_option_home_nosuid
-    - mount_option_home_nodev
+    - mount_option_home_noexec
 
     # /proc hidepid = 1 Contains process information and the system
+
```
Comment (jan-cerny) [inline controls/anssi.yml:194]: What about this? Will there be a rule for it?

Comment (yuumasato) [reply controls/anssi.yml:194]: Nice catch, there should be a rule for it, and actually, it should `hidepid=2`

**Comment by yuumasato** [comment]
Closing, this is part of upstream already.

**Pull request closed without merging**